packages/client/subscriptions/MeetingSubscription.ts

```diff
@@ -189,21 +190,4 @@ const updateHandlers = {
   SetMeetingMusicSuccess: setMeetingMusicMeetingUpdater
 } as const
 
-const MeetingSubscription = (
-  atmosphere: Atmosphere,
-  variables: MeetingSubscription$variables,
-  router: {history: RouterProps['history']}
-) => {
-  atmosphere.registerSubscription(subscription)
-  return requestSubscription<TMeetingSubscription>(atmosphere, {
-    subscription,
-    variables,
-    updater: subscriptionUpdater('meetingSubscription', updateHandlers, atmosphere),
-    onNext: subscriptionOnNext('meetingSubscription', onNextHandlers, atmosphere, router),
-    onCompleted: () => {
-      atmosphere.unregisterSub(MeetingSubscription.name, variables)
```

Comment:
This was wrong, l208 explicitly set the key to be independent of minified function names while this line still depends on it. When minified it would also not match the operation name.